tasks/build_tags.py

```diff
@@ -78,14 +78,23 @@ def get_default_build_tags(puppy=False, process=False):
     if not process:
         exclude = exclude + PROCESS_ONLY_TAGS
 
-    # remove all tags that are only available on debian distributions
-    distname = distro.id().lower()
-    if distname not in REDHAT_DEBIAN_SUSE_DIST:
-        exclude = exclude + REDHAT_DEBIAN_SUSE_ONLY_TAGS
+    # remove all tags that are not available for current distro
+    exclude.extend(get_distro_exclude_tags())
```

Comment:
Is this being used in more places than before? Asking because not all builder images might have the `distro` dependency installed.

Reply:
Not sure to understand your question. The behaviour should be same as before, if the distro doesn't match, `get_distro_exclude_tags` will just return an empty list.

Reply:
👍 Nothing changed in the use of `distro` indeed.